public/pages/Snapshots/components/SnapshotRestoreAdvancedOptions/SnapshotRestoreAdvancedOptions.tsx

```diff
@@ -6,9 +6,12 @@
 import React, { ChangeEvent } from "react";
 import { EuiCheckbox, EuiSpacer, EuiText } from "@elastic/eui";
 import CustomLabel from "../../../../components/CustomLabel";
+import IndexSettingsInput from "../../components/IndexSettingsInput";
```

Comment:
Updates in this file, once again, are to enable user input to override or ignore index settings during a snapshot restore.  This happens from within the Advanced Options menu (this component)